_inc/lib/admin-pages/class-jetpack-about-page.php

```diff
@@ -0,0 +1,676 @@
+						</p>
+						<p>
+							<?php esc_html_e( 'We believe in Open Source and the vast majority of our work is available under the GPL.', 'jetpack' ); ?>
+						</p>
+						<p>
+							<?php
+								// Maybe use printf() because we'll want to escape the string but still allow for the link, so we can't use esc_html_e().
+								echo wp_kses(
+									__( 'We strive to live by the <a href="https://automattic.com/creed/" target="_blank" class="jptracks" data-jptracks-name="jetpack_about_creed" rel="noopener noreferrer">Automattic Creed</a>.', 'jetpack' ),
+									array(
+										'a' => array(
+											'href'   => array(),
+											'class'  => array(),
+											'target' => array(),
+											'rel'    => array(),
+											'data-jptracks-name' => array(),
+										),
+									)
+								);
+							?>
+						</p>
+						<p>
+							<a href="https://automattic.com/work-with-us" target="_blank" rel="noopener noreferrer" class="jptracks" data-jptracks-name="jetpack_about_work_with_us">
+								<?php esc_html_e( 'Come work with us', 'jetpack' ); ?>
+							</a>
+						</p>
+					</div>
+				</div>
+			</div>
+
+			<div class="jetpack-about__colophon">
+				<h3><?php esc_html_e( 'Popular WordPress services by Automattic', 'jetpack' ); ?></h3>
+				<ul class="jetpack-about__services">
+				<?php $this->display_plugins(); ?>
+				</ul>
+
+				<p class="jetpack-about__services-more">
+				<?php
+				echo wp_kses(
+					__( 'For even more of our WordPress plugins, please <a href="https://profiles.wordpress.org/automattic/#content-plugins" target="_blank" rel="noopener noreferrer" class="jptracks" data-jptracks-name="jetpack_about_wporg_profile">take a look at our WordPress.org profile</a>.', 'jetpack' ),
+					array(
+						'a' => array(
+							'href'               => array(),
+							'target'             => array(),
+							'rel'                => array(),
+							'class'              => array(),
+							'data-jptracks-name' => array(),
+						),
+					)
+				);
+				?>
+														</p>
+			</div>
+		</div>
+		<?php
+	}
+
+	/**
+	 * Add information cards for a8c plugins.
+	 */
+	public function display_plugins() {
+		$plugins_allowedtags = array(
+			'a'       => array(
+				'href'   => array(),
+				'title'  => array(),
+				'target' => array(),
+			),
+			'abbr'    => array( 'title' => array() ),
+			'acronym' => array( 'title' => array() ),
+			'code'    => array(),
+			'pre'     => array(),
+			'em'      => array(),
+			'strong'  => array(),
+			'ul'      => array(),
+			'ol'      => array(),
+			'li'      => array(),
+			'p'       => array(),
+			'br'      => array(),
+		);
+
+		// slugs for plugins we want to display.
+		$a8c_plugins = array(
+			'woocommerce',
+			'wp-super-cache',
+			'wp-job-manager',
+			'co-authors-plus',
```

Comment:
Likewise, would be good to make this something we can edit on our end. Non-blocking.